network/a10/a10_server_axapi3.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Ansible module to manage A10 Networks slb server objects
+(c) 2014, Mischa Peters <mpeters@a10networks.com>, 2016, Eric Chou <ericc@a10networks.com>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: a10_server_axapi3
+version_added: 2.3
+short_description: Manage A10 Networks AX/SoftAX/Thunder/vThunder devices
+description:
+    - Manage slb server objects on A10 Networks devices via aXAPIv3
+author: "Eric Chou (@ericchou) based on previous work by Mischa Peters (@mischapeters)"
+options:
+  host:
+    description:
+      - hostname or ip of your A10 Networks device
+    required: true
+  username:
+    description:
+      - admin account of your A10 Networks device
+    required: true
+    aliases: ['user', 'admin']
+  password:
+    description:
+      - admin password of your A10 Networks device
+    required: true
+    aliases: ['pass', 'pwd']
+  server_name:
+    description:
+      - slb server name
+    required: true
+    aliases: ['server']
+  server_ip:
+    description:
+      - slb server IP address
+    required: false
+    default: null
+    aliases: ['ip', 'address']
+  server_status:
```

Comment:
Hi @resmo, I see your point. I aim to have this part match up with what people would see in the cli via 'show json-config' that we added in AXAPIv3 to allow people to construct the JSon body of the API call. So essentially, this is what people would see for the test case I listed out in summary: 

TH3030-2#sh json-config slb server

a10-url:/axapi/v3/slb/server
{
  "server-list": [
    {
      "name":"TEST_1",
      "host":"1.1.1.100",
      **"action":"enable"**,
      "uuid":"49dff164-9b75-11e6-8613-e3611c8ca89d",
      "port-list": [
        {
          "port-number":8080,
          "protocol":"tcp",
          **"action":"enable"**,
          "uuid":"49dff696-9b75-11e6-8613-e3611c8ca89d"
        },
        {
          "port-number":8443,
          "protocol":"tcp",
          "action":"enable",
          "uuid":"49dffbbe-9b75-11e6-8613-e3611c8ca89d"
        }
      ]
    }
  ]
}
TH3030-2#

I thought about just making it 'action' but thought it is even less descriptive. Since in the previous version a10_server.py they use 'server_status' I settled on the alias of using either 'server_status' if you used to the syntax or 'action' if you conform to our new term. 

Let me know what you think. Thank you.